apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/addEditFormDefinitionEditor.tsx

```diff
@@ -135,9 +133,9 @@ const ClientRole = ({ roleNames, clientId, anonymousRead, configuration, onUpdat
       anonymousRead={anonymousRead}
       roleSelectFunc={onUpdateRoles}
       nameColumnWidth={40}
-      service="FileType"
+      service="FormService"
       checkedRoles={[
-        { title: types[0].name, selectedRoles: configuration[types[0].type] },
+        { title: types[0].name, selectedRoles: configuration[types[0].type], disabled: anonymousRead },
```

Comment:
is the disabled anonymousRead a typo in the RoleTable?

Reply:
Hi @thdlam, the disabledRead is an existing property the role table.